scaffolds_actor_model/elixir/lib/actor_model/person.ex

```diff
@@ -0,0 +1,58 @@
+defmodule ActorModel.Person do
+  require Logger
+
+  @enforce_keys [:name, :friend_name]
+
+  defstruct friend_name: nil, name: nil, count_greetings_sent: 0, count_greetings_received: 0
+
+  @type t() :: %__MODULE__{
+          name: atom(),
+          friend_name: Genserver.server(),
```

Comment:
From the example below, it looks like `friend_name` is also an atom 🤔

Reply:
(Yup, it's one of the possibilities for `Genserver.server()`)

See: https://hexdocs.pm/elixir/GenServer.html#t:server/0

```
server() :: pid() | name() | {atom(), node()}
```
and then:

```
name() :: atom() | {:global, term()} | {:via, module(), term()}
```

Basically we need to do a GenServer.whereis/2 with it, so it should be a type accepted by that function.

(In the tests, we use a PID directly.)